var/spack/repos/builtin/packages/py-ruff/package.py

```diff
@@ -15,8 +15,8 @@ class PyRuff(PythonPackage):
 
     license("MIT")
 
+    version("0.3.0", sha256="0886184ba2618d815067cf43e005388967b67ab9c80df52b32ec1152ab49f53a")
     version("0.1.6", sha256="1b09f29b16c6ead5ea6b097ef2764b42372aebe363722f1605ecbcd2b9207184")
-    version("0.0.276", sha256="d456c86eb6ce9225507f24fcc7bf72fa031bb7cc750023310e62889bf4ad4b6a")
```

Comment:
Is there a reason to remove versus deprecate this version?  See docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions for guidelines.

Reply:
Well, we could say this package produces tags at a very high rate: https://github.com/astral-sh/ruff/tags so if we measure the distance in number of tags it's a very old one. Other reasons could be there is no package that depends on this one and it's unlikely there is a good reason to build an older version other than incompatibilities with rust in the versions that no one is tracking. But I can mark it as deprecated instead